parastell/nwl_utils.py

```diff
@@ -5,8 +5,10 @@
 import pystell.read_vmec as read_vmec
 import matplotlib.pyplot as plt
 import concurrent.futures
-from multiprocessing import cpu_count
```

Comment:
Why are we removing `multiprocessing`? Wouldn't that remove parallelization from the NWL routine?

Reply:
`concurrent.futures` is what does the multiprocesssing, I originally attempted to make some guess about how many cpus to use based on `cpu_count` but ended up removing that bit, but this import remained